### PR TITLE
Cleanup in module `xcube.core.varexpr`

### PR DESCRIPTION
**Pull request by forman**
Cleanup in module `xcube.core.varexpr`

Checklist:

* [ ] ~Add unit tests and/or doctests in docstrings~
* [ ] ~Add docstrings and API docs for any new/modified user-facing classes and functions~
* [ ] ~New/modified features documented in `docs/source/*`~
* [ ] ~Changes documented in `CHANGES.md`~
* [x] GitHub CI passes
* [x] AppVeyor CI passes
* [x] Test coverage remains or increases (target 100%)

### CODE REVIEW

File: xcube/core/varexpr/context.py [hunk 1]
```diff
@@ -171,6 +154,21 @@ def evaluate(self, var_expr: str) -> xr.DataArray:
         return result
 
 
-class VarExprValidator(ast.NodeTransformer):
-    def visit_Lambda(self, node):
-        raise VarExprError("lambda expressions are not supported")
+def _format_callable(name: str, fn: Union[np.ufunc, Callable]):
```
Comment (konstntokas) [inline xcube/core/varexpr/context.py:157]: What is the reason for putting this outside of the class?

**Review by konstntokas — APPROVED** [review]
Yes looks cleaner than before. Would you like to update CHANGES.md?

**Comment by forman** [comment]
> Would you like to update CHANGES.md?

No, this code and the former has not been released yet.

File: xcube/core/varexpr/context.py (hunk 1 above)
Comment (forman) [reply xcube/core/varexpr/context.py:157]: It is was a private static method and now it is a private function, calling is shorter as it does not require context object.

**Pull request merged**